worlds/dkc3/Rom.py

```diff
@@ -0,0 +1,547 @@
+import Utils
+from Patch import read_rom
+from .Locations import lookup_id_to_name, all_locations
+from .Levels import level_list, level_dict
+
+USHASH = '120abf304f0c40fe059f6a192ed4f947'
+ROM_PLAYER_LIMIT = 65535
+
+import hashlib
+import os
+import math
+
+
+location_rom_data = {
+    0xDC3000: [0x657, 1],
```

Comment:
Can the keys of this dict be written in symbolic form? I think that would be more readable (can always keep the hex value in a comment if it's handy to still be here). Just at a glance it kind of looked as if hex data was copied from the ROM to here, though I see it really wasn't. These values will already have names in the data package, right?

Reply:
I could do something like that, though doing so would either necessitate
1) just storing what is now the key along with the values, or
2) doing a lookup of every single location name to ID on every SNI tick, which would not be ideal. I'll consider adding comments to be descriptive of which locations are which, for readability.

Reply:
Hmm. Can you change the line to something along the lines of

`Locations.level_location_table["Lakeside Limbo - Flag"]: [0x657, 1], # location id 0xDC3000`

where the key evaluates to 0xDC3000 before going into the dict? I'm thinking it would happen only once and not necessitate any changes to other code.

Reply:
Hm, unless there is a reason to rework this, I would probably simply put a comment next to each pair saying what it is.
Ideally you could try to generate that dict from a bigger dataset (that includes names) with a generator and make sure the whole dataset is always backwards-compatible (old seed with new client, or old seed with new datapackage).

Reply:
I intend to document meaning via comments in this case.